sdk/iot/doc/how_to_iot_hub_samples_windows.md

```diff
@@ -0,0 +1,395 @@
+    It will be used to create the logical device on your Azure IoT Hub.
+
+    > For Windows, follow the instruction in the output of the script and download the Baltimore PEM CA from https://www.digicert.com/digicert-root-certificates.htm into the same place mentioned by the script output.
+
+    You should have it saved as shown bellow:
+
+    ```shell
+    C:\azure-sdk-for-c\sdk\iot\hub\samples\src>dir BaltimoreCyberTrustRoot.crt.pem
+    Volume in drive C is OSDisk
+    Volume Serial Number is AAAA-BBBB
+
+    Directory of C:\azure-sdk-for-c\sdk\iot\hub\samples\src
+
+    06/03/2020  03:33 PM             1,262 BaltimoreCyberTrustRoot.crt.pem
+                1 File(s)          1,262 bytes
+    ...
+    ```
+
+06. Create a logical device
+
+    - Log into your Azure account on [Azure Portal](https://portal.azure.com)
+    - On the menu on the left, click on "IoT devices" under "Explorers".
+    - Click on "New".
+    - Type an unique ID for your device.
+    - Select "X.509 Self-Signed" for "Authentication type". 
+    - Type the fingerprint obtained in the previous step (the same can be used for primary and secondary Thumbprints; no colons!).
+    - Click on "Save".
+
+07. Collect information about Azure IoT Hub and device
+
+    For the Azure IoT Embedded SDK for C samples, we will need the Azure IoT Hub name and device ID.
+
+    - Get the Azure IoT Hub FQDN.
+        - On your Azure IoT Hub page, click on "Overview".
+        - Copy and save the "Hostname" value (in this example, "myiothub.azure-devices.net").
+    - Get the device ID. 
+    - On your Azure IoT Hub page, click on "IoT devices" under "Explorers".
+    - On the list of devices, click on the device created on the previous step (in this example, "paho-sample-device1").
+    - Copy and save the "Device ID" value (in this example, "paho-sample-device1"). 
+
+
+08. Set the environment variables needed for the samples
+
+    According the the [readme documentation](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/iot/hub/samples) for the Azure Embedded SDK for C IoT client samples require the following environment variables.
+
+    ```shell
+    set AZ_IOT_DEVICE_ID=<device ID obtained on step 7> 
+    set AZ_IOT_HUB_HOSTNAME=<FQDN obtained on step 7>
+    ```
+
+    AZ_IOT_DEVICE_X509_CERT_PEM_FILE and AZ_IOT_DEVICE_X509_TRUST_PEM_FILE have already been set on step 5.
+
+    Using the values from the example above, the export command would look like this (don't run these command lines, you should use your own device ID and hostname)
+
+    ```shell
+    C:\azure-sdk-for-c>set AZ_IOT_DEVICE_ID=paho-sample-device1
+    C:\azure-sdk-for-c>set AZ_IOT_HUB_HOSTNAME=myiothub.azure-devices.net
+    ```
+
+09. Create and open the solution for the Azure Embedded SDK for C
+
+    Back into the folder where the Azure SDK for C was cloned...
+
+    ```shell
+    C:\azure-sdk-for-c>mkdir cmake
+    C:\azure-sdk-for-c>cd cmake
+    C:\azure-sdk-for-c\cmake>cmake ..
```

Comment:
This needs `-DTRANSPORT_PAHO=ON` right?

Reply:
Sorry for the late comment. I was double checking here that the `-A Win32` was removed if we are going `x64-windows-static`

Reply:
MIA

Reply:
Fixed.